Makefile

```diff
@@ -24,14 +24,16 @@ remote-resource-visibility = https://github.com/qld-gov-au/ckanext-resource-visi
 develop-resource-visibility = https://github.com/qld-gov-au/ckanext-resource-visibility tag develop
 
 remote-validation = https://github.com/qld-gov-au/ckanext-validation.git tag  v0.0.8-qgov.5
-develop-validation = https://github.com/DataShades/ckanext-validation.git branch develop
+develop-validation = https://github.com/qld-gov-au/ckanext-validation.git branch develop
 
 remote-ytp-comments = https://github.com/qld-gov-au/ckanext-ytp-comments.git tag  2.5.0-qgov.7
+develop-ytp-comments = https://github.com/qld-gov-au/ckanext-ytp-comments.git branch develop
+
 remote-datajson = https://github.com/GSA/ckanext-datajson.git branch main
 remote-xloader = https://github.com/qld-gov-au/ckanext-xloader.git tag 0.10.0-qgov.1
 remote-csrf-filter = https://github.com/qld-gov-au/ckanext-csrf-filter.git tag 1.1.5
 
-remote-datarequests = https://github.com/qld-gov-au/ckanext-datarequests.git tag 2.2.1-qgov.3
+remote-datarequests = https://github.com/qld-gov-au/ckanext-datarequests.git tag 2.2.1-qgov.8
```

Comment:
It's worrying to me that this Makefile is apparently still being actively used and relied on, when it's not a part of our standard tested build process. Having this file seems like it would encourage building things that _appear_ to work and ignoring the fact that they don't pass CI testing.

Also, if you really want to install on a local machine, running `bin/init.sh` and `bin/init-ext.sh` should get you 99% of the way there.